### ISSUE TITLE
Re-order test run sequence

### ISSUE THREAD

**Issue by Onokaev** [microsoftgraph/microsoft-graph-explorer-v4#2355]
Currently, when UI tests fail, accessibility tests are skipped. We need to re-order the tests so that accessibility tests run regardless of the UI tests failing
